h/static/scripts/polyfills.js

```diff
@@ -24,6 +24,10 @@ try {
   require('js-polyfills/url');
 }
 
+// KeyboardEvent.prototype.key
+// (Native in Chrome >= 51, Firefox >= 23, IE >= 9)
+require('keyboardevent-key-polyfill');
```

Comment:
KeyboardEvent.prototype.key is the modern standards-based way to get the character or special key that was pressed. It is available for both 'keydown' and 'keypress' events. Unfortunately it was only recently supported in Chrome and is still not supported in Safari. Fortunately it looks like a basic polyfill is quite small.